src/FamilyTree.jsx

```diff
@@ -0,0 +1,179 @@
+      <div 
+        className = {className} 
+        onMouseEnter = {this.handleMouseHover}
+        onMouseLeave = {this.handleMouseHover}    
+      >
+        <p className="label-text">{nodeData.attributes.node_name}</p>
+        {
+          this.state.isHovering && // The below info will show up on hover
+          <div class = "PopUp">
+          <BitByteLabel
+            id = {nodeData.attributes.id}
+            node_name = {nodeData.attributes.node_name}
+            tree_name = {nodeData.attributes.tree_name}
+            quarter_joined = {nodeData.attributes.quarter_joined}
+            college = {nodeData.attributes.college}
+            major = {nodeData.attributes.major}
+            class_year = {nodeData.attributes.class_year}
+            linkedin = {nodeData.attributes.linkedin}
+            facebook = {nodeData.attributes.facebook}
+            instagram = {nodeData.attributes.instagram}
+          />
+          </div>
+        }
+      </div>
+    )
+  }
+}
+
+/* Hard-coded data below */
+const bitByteTreeData = [
+  // Byte
+  {
+    name :'Waggie Mao',
+    
+    attributes:{
+      id: 12345678,
+      node_name : "Waggie Wao",
+      tree_name : "Bug - free tree",
+      quarter_joined : "WI 20",
+      college : "Sixth",
+      major : "Computer Science",
+      class_year : 2022,
+      linkedin : "https://linkedin.com/in/YiyueMaggieMao",
+      facebook : "https://www.facebook.com/YiyueMaggieMao",
+      instagram : "https://www.instagram.com/maggie_maoph/"
+    },
+
+    children:[
+      // Bit 1
+      {
+        name : 'Judy Liu',
+        attributes:{
+          id: 12345679,
+          node_name : "Judy Liu",
+          tree_name : "Bug - free tree",
+          quarter_joined : "WI 20",
+          college : "Warren",
+          major : "Math-CS",
+          class_year : 2023,
+          linkedin : "https://linkedin.com/in/YiyueMaggieMao",
+          facebook : "https://www.facebook.com/YiyueMaggieMao",
+         instagram : "https://www.instagram.com/maggie_maoph/"
+        },
+      },
+
+      // Bit 2
+      {
+        name : 'Tiffany Chang',
+        attributes:{
+          id: 12345680,
+          node_name : "Tiffany Chang",
+          tree_name : "Bug - free tree",
+          quarter_joined : "WI 20",
+          college : "Warren",
+          major : "Computer Science",
+          class_year : 2023,
+          linkedin : "https://linkedin.com/in/YiyueMaggieMao",
+          facebook : "https://www.facebook.com/YiyueMaggieMao",
+         instagram : "https://www.instagram.com/maggie_maoph/"
+        },
+      },
+    ],
+  },
+];
+
+/** 
+ * Component returning a div containing a tree
+ * Need to declare as class for ComponentDidMount() 
+ */
+class FamilyTree extends React.PureComponent {
+
+    state={}
+
+    /* Used to center the tree on render */
+    componentDidMount() {
+      const dimensions = this.treeContainer.getBoundingClientRect();
+      console.log(dimensions.width);
```

Comment:
Remove console logs in production

Reply:
Removed, will be pushed with other commits upon next push